### DISCUSSION TITLE
Help with changing the theme

### DISCUSSION THREAD

**Discussion by Lucas31102003** [chakra-ui/chakra-ui#9093] [Q&A] [answered]
Hey guys!

I have a problem in my project where I can't set the theme to light. I tried to use ColorModeProvider or something like that, but in the html tag it takes the class="dark" and a style in line as color-scheme: dark line;

Can you help me ❤️

```
const Main = () => {
  return (
    <Theme appearance="light">
      <Routes>
        <Route path="/home" element={<Home />} />
        <Route path="/mktp/data" element={<Data />} />
        <Route path="/mktp/application" element={<Application />} />
        <Route path="/mktp/ml" element={<MachineLearning />} />
        <Route path="/adm/users" element={<Users />} />
        <Route path="/adm/objects" element={<Objects />} />
      </Routes>
    </Theme>
  );
};
ReactDOM.createRoot(document.getElementById("root")).render(
  <>
    <Router>
      <Provider>
        <Main />
      </Provider>
    </Router>
  </>
);
```

![Screenshot_1](https://github.com/user-attachments/assets/78cb6140-d156-44b0-bec5-7fa924d1d9c2)

**Comment by segunadebayo** [comment]
You should consider clearing your local storage if it's set.

Alternatively, you can set the `forcedTheme` prop on the `ColorModeProvider` in `components/ui/provider.tsx`

```tsx
<ColorModeProvider forcedTheme="light">
 {props.children}
</ColorModeProvider>
```

**Comment by LucasPirolli** [comment]
Hello Segun! It was on my old GitHub account. 
Just stopping by to say thanks for the previous insights. I applied the resolution you recommended to me in the `components/ui/provider.tsx` file and modified it to:

```
export function Provider(props) {
  return (
    <ChakraProvider Value={defaultSystem}>
      <ColorModeProvider forcedTheme="light">{props.children}</ColorModeProvider>
    </ChakraProvider>
  )
}
```

Appreciated for the help ❤️

**Comment by segunadebayo** [comment, marked as answer]
`html` stays `dark` when `next-themes` has a stored value. `forcedTheme="light"` on `ColorModeProvider` wins.

Clear localStorage if you want the toggle back.
